src/furo/assets/styles/_layout.scss

```diff
@@ -271,6 +271,11 @@ body {
         width: 1.25rem;
       }
     }
+    .header-center {
+      a {
+        text-decoration: none;
```

Comment:
Could you move this to _theme.scss? Since, uhm... That's where it fits in my mental model which I still have to document.

Reply:
Updated! Also included the `color:` I forgot to push the first time.